9c-internal/multiplanetary/network/heimdall.yaml

```diff
@@ -159,7 +171,7 @@ remoteHeadless:
     data: 500Gi
 
   hosts:
-  - "heimdall-internal-rpc-1.nine-chronicles.com"
+  - "heimdall-internal-rpc-1.nine-chronicles.com,heimdall-preview-rpc-1.nine-chronicles.com"
```

Comment:
would headless host argument accept this format of string? i think it might be needed to be sanitized in helm templates where using headless